ui_framework/components/local_nav/_local_search.scss

```diff
@@ -28,6 +28,10 @@
   }
 }
 
+.kuiLocalLuceneSearchInput:focus {
+  padding-right: 13em;
+}
```

Comment:
I'd characterize this as a custom class, specific to the way we're using the LocalSearch components in Discover and elsewhere in Kibana. Looking at it from that perspective, I think it's probably best to rename this to something like `localLuceneSearchInput` (reserving the kui prefix for the UI Framework classes), and put it in `src/core_plugins/kibana/public/kibana.less` (or somewhere within that directory).

I think the counter-point is that we want to use this help link in every instance of the LocalSearch, but I think we should still draw a line and use the UI Framework to provide generic components and guidance on patterns for using them, and use `src` to define more concrete rules on how these generic components are used in Kibana. What do you think @snide ?